Xwt.XamMac/Xwt.Mac/PopoverBackend.cs

```diff
@@ -305,7 +305,7 @@ public class NSAppearanceCustomizationPopover : NSPopover, INSAppearanceCustomiz
 			public NSAppearanceCustomizationPopover ()
 			{ }
 
-			protected NSAppearanceCustomizationPopover (IntPtr handle) : base (handle)
+			protected NSAppearanceCustomizationPopover (NativeHandle handle) : base (handle)
```

Comment:
These `NativeHandle` changes mean we cannot merge this until we're ready to build VSMac with a newer macos workload.